data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/log/SingleFileImportLogger.java

```diff
@@ -0,0 +1,230 @@
+package com.scalar.db.dataloader.core.dataimport.log;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.scalar.db.dataloader.core.dataimport.datachunk.ImportDataChunkStatus;
+import com.scalar.db.dataloader.core.dataimport.log.writer.LogWriter;
+import com.scalar.db.dataloader.core.dataimport.log.writer.LogWriterFactory;
+import com.scalar.db.dataloader.core.dataimport.task.result.ImportTargetResult;
+import com.scalar.db.dataloader.core.dataimport.task.result.ImportTargetResultStatus;
+import com.scalar.db.dataloader.core.dataimport.task.result.ImportTaskResult;
+import com.scalar.db.dataloader.core.dataimport.transactionbatch.ImportTransactionBatchResult;
+import java.io.IOException;
+import javax.annotation.concurrent.ThreadSafe;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * An implementation of {@link AbstractImportLogger} that uses a single file for each log type.
+ * Unlike {@link SplitByDataChunkImportLogger}, this logger creates only three log files: one for
+ * successful operations, one for failed operations, and one for summary information, regardless of
+ * the number of data chunks processed.
+ *
+ * <p>The log files are named as follows:
+ *
+ * <ul>
+ *   <li>success.json - Records of successful import operations
+ *   <li>failure.json - Records of failed import operations
+ *   <li>summary.log - Summary information for all data chunks
+ * </ul>
+ */
+@ThreadSafe
+public class SingleFileImportLogger extends AbstractImportLogger {
+
+  protected static final String SUMMARY_LOG_FILE_NAME = "summary.log";
+  protected static final String SUCCESS_LOG_FILE_NAME = "success.json";
+  protected static final String FAILURE_LOG_FILE_NAME = "failure.json";
+  private static final Logger LOGGER = LoggerFactory.getLogger(SingleFileImportLogger.class);
```

Comment:
According to the Google Java Style Guide, a logger is not considered a constant and should not use UPPER_SNAKE_CASE:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

```suggestion
  private static final Logger logger = LoggerFactory.getLogger(SingleFileImportLogger.class);
```

If other classes also need to be updated, you can handle them in a separate PR.

Reply:
I will update the files related to this PR, in this PR itself, for others I will create a separate PR.
Thank you.